src/other/PropertyEditor.js

```diff
@@ -78,6 +93,49 @@
         }
         return this.show_settings() ? [this] : this.widget() ? [this.widget()] : [];
     };
+    PropertyEditor.prototype.rootBoderRadius = function () {
+        var radiusStyleObj = null;
+        var radiusValExist = false;
+        if(this.borderRadiusAll_exists() && this.borderRadiusAll()){
+             radiusStyleObj = {};
+             radiusStyleObj['border-radius'] = this.borderRadiusAll()+'em';
+             radiusValExist = true;
+         }
+         if(this.borderRadiusTop_exists() && this.borderRadiusTop() ){
+             radiusStyleObj = {};
+             radiusStyleObj['border-top-left-radius'] = this.borderRadiusTop()+'em';
+             radiusStyleObj['border-top-right-radius'] = this.borderRadiusTop()+'em';
+             radiusValExist = true;
+         }
+        if(this.borderRadiusTopLeft_exists() && this.borderRadiusTopLeft()){
+            radiusStyleObj = {};
+            radiusStyleObj['border-top-left-radius'] = this.borderRadiusTopLeft()+'em';
+            radiusValExist = true;
+        }
+
+        if(this.borderRadiusTopRight_exists() && this.borderRadiusTopRight()){
+            radiusStyleObj = {};
+            radiusStyleObj['border-top-right-radius'] = this.borderRadiusTopRight()+'em';
+            radiusValExist = true;
+        }
+        if(this.borderRadiusBottom_exists() && this.borderRadiusBottom()){
+            radiusStyleObj = {};
+            radiusStyleObj['border-bottom-right-radius'] = this.borderRadiusBottom()+'em';
+            radiusStyleObj['border-bottom-left-radius'] = this.borderRadiusBottom()+'em';
+        }
+        if(this.borderRadiusBottomRight_exists() && this.borderRadiusBottomRight()){
+            radiusStyleObj = {};
+            radiusStyleObj['border-bottom-right-radius'] = this.borderRadiusBottomRight()+'em';
+            radiusValExist = true;
+        }
+        if(this.borderRadiusBottomLeft_exists() && this.borderRadiusBottomLeft()){
+            radiusStyleObj = {};
+            radiusStyleObj['border-bottom-left-radius'] = this.borderRadiusBottomLeft()+'em';
+            radiusValExist = true;
+        }
+        return radiusStyleObj;
+    };
+
```

Comment:
Your JS probably handles it but it looks here like you might set the shorthand border-radius and then possibly expect to use individual inherited properties, which my understanding would be ignored by the browser? If it could be an issue perhaps some logic here to avoid that situation.